Mathlib/LinearAlgebra/LinearPMap.lean

```diff
@@ -758,6 +755,53 @@ theorem domRestrict_le {f : E →ₗ.[R] F} {S : Submodule R E} : f.domRestrict
   ⟨by simp, fun x y hxy => domRestrict_apply hxy⟩
 #align linear_pmap.dom_restrict_le LinearPMap.domRestrict_le
 
+/-- The domain of the composition of two `LinearPMap`s is `{x : f.domain | f x ∈ g.domain }`. -/
+def comp_domain (g : F →ₗ.[R] G) (f : E →ₗ.[R] F) : Submodule R E where
+  carrier := { x | ∃ y : f.domain, x = y ∧ f y ∈ g.domain }
```

Comment:
Would it be worth defining `Submodule.Pcomap` that does this kind of things?

Reply:
I don't know. At the moment, I think I don't need anything more about the composition. I want to prove T^{-1} - S^{-1} = T^{-1} (S - T) S^{-1} and for that the contents of this PR should be enough.